x-pack/plugins/security_solution/public/cases/components/all_cases/index.tsx

```diff
@@ -82,7 +82,7 @@ const getSortField = (field: string): SortFieldCase => {
 };
 
 interface AllCasesProps {
-  onRowClick?: (id?: string) => void;
+  onRowClick?: (theCase?: Case) => void;
```

Comment:
I do not understand why changing  from caseId to the full object of case because at the end we are only using the id

Reply:
Because here https://github.com/elastic/kibana/pull/82996/files?file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx&hide-deleted-files=true#diff-a42e73bb6513fe4fb034631921bdb628c760b2162c6a600d36964d738238dab1R51 I wanted the title of the case. I can change it and pass only the id and the title. What do you think? I agree, passing the full object is too much :)

Reply:
I changed it to pass the title along with the id.

Reply:
ok I did not see that, make sense to me now, thanks for explanation! LGTM on that